airbyte-integrations/connectors/destination-s3-v2/gradle.properties

```diff
@@ -0,0 +1 @@
+testExecutionConcurrency=-1
```

Comment:
I don't understand what changed, but this used to not be necessary https://airbytehq-team.slack.com/archives/C06E6CQMDT3/p1731520778573749

Reply:
Did we upgrade gradle?

Reply:
:shrug: I didn't bother digging into the commit history for this, the workaround is easy enough